sdk/python/kfp/client/client.py

```diff
@@ -760,6 +760,51 @@ def run_pipeline(
             IPython.display.display(IPython.display.HTML(html))
         return response.run
 
+    def archive_run(self, run_id: str):
+        """Archives a run.
+
+        Args:
+            run_id: id of the run.
+
+        Returns:
+            Object. If the method is called asynchronously, returns the request
+            thread.
+
+        Raises:
+            kfp_server_api.ApiException: If the run is not found.
+        """
+        return self._run_api.archive_run(run_id=run_id)
```

Comment:
`.archive_run` takes an `id` parameter rather than a `run_id` parameter, I think.

Same for `.unarchive_run`.

Reply:
Yikes. Good catch, thanks! 🙃